src/txn/txn.c

```diff
@@ -1277,7 +1287,8 @@ __txn_commit_timestamps_assert(WT_SESSION_IMPL *session)
             op_timestamp = txn->commit_timestamp;
         if (F_ISSET(txn, WT_TXN_TS_COMMIT_KEYS) && op_timestamp < prev_op_timestamp)
```

Comment:
So we allow commit timestamp to be out of order?

Reply:
In history store, the keys are ordered by commit timestamp not durable timestamp.

Reply:
They've asked for a `durable_timestamp=mixed_mode` setting.

Reply:
I'll check whether that's what they actually wanted.